### ISSUE TITLE
Major accessibility issues in 8.8

### ISSUE THREAD

**Issue by Elshara** [UnigramDev/Unigram#2777]
**Describe the bug**
Ever since Unigram_8.6.7045, the interface has been getting unusable continuously. And it is an accessibility bottleneck. I have been a loyal user of Unigram since probably Unigram V5 if not earlier. At least since 2018. This app is the only thing keeping me on Windows, period. And in the latest version Unigram_8.8.7433.0_x64, chats are completely inaccessible! Contacts, messages, object navigation, leads nowhere. The main reason why I keep in contact with people on Telegram, you disabled accessibility support. I don't know if it's a bug, or if you're just messing up the interface on purpose. but it is absolutely uncalled for and I am about ready to jump ship to Matrix. Say what you want about Element, at least it is accessible.
**To Reproduce**
1. Go to ' unigramapps channel'
2. Click on 'the latest unigram update file'
3. Scroll down to 'the installer and do that'
4. In screen readers, all versions, chats are completely silent.

**Expected behavior**
This should never have happened. It should never have been released either to the store, or to the channel. Do you test anything before it goes live to everyone?
**Screenshots**
I would add a tutorial, but that would just be kicking the bucket as in, completely and utterly redundant.
**Version Info**
 - Unigram: [8.8.7433.0_x64]
 - Windows: [11 Version 22H2 build 25145.1000]

**Additional context**
I still appreciate the work you guys put into making telegram accessible on windows laptops and desktops. You will always be a company that I admire for your hard work. And I would seriously, absolutely be lost without your guidance. Especially now days. Please, I'm begging you. Fix and stabilize your code before you push it out to people. This app is where all my friends and community interests are at. And in one place, its existence has changed everything for me. I would even pay you to fix this. Unigram is a life saver. I am already a telegram premium subscriber. Whatever you need, I am available to test. I just do not like the updates since the new interface came out. I need a stable messaging platform.

**Comment by FrayxRulez** [comment]
@Elshara No, I never test the software for accessibility honestly, I have no time for that.

**Comment by FrayxRulez** [comment]
@Elshara But I’ll definitely fix the issue ASAP

**Comment by Elshara** [comment]
Thank you! <3 I am using Unigram_8.6.7045.0_x64 for now until all these bugs can be fixed. I'm, more familiar with the layout of that version and it is the most updated with the old style menus. I appreciate what you're doing, I just don't think it's ready for screen reader users right now.

**Comment by MarcAnt01** [comment]
Anyway, if you want a stable version, I would suggest you to stick to the Microsoft Store, since out-of-the-store builds are betas

**Comment by AlexandrKozlovskiy** [comment]
@FrayxRulez I have the same accessibility bug. it still not fixed. Also please do accessible results of vote in pols.

**Comment by FrayxRulez** [comment]
@AlexandrKozlovskiy Fixing stuff takes time, and release takes even more time. The update should be available later today.

**Comment by amirmahdifard** [comment]
@AlexandrKozlovskiy not only that one, lots of other big problem is there.

**Comment by AlexandrKozlovskiy** [comment]
@FrayxRulez Can you please send to me privately compiled new version of telegram with accessibility fixes (you know my tg).

**Comment by Simon818** [comment]
I know this comment is not strictly necessary but I wanted to say thanks for such a quick fix on this. Unigram is truly one of the only accessible clients left. iOS has a giant backlog of issues that nobody seems to care enough to fix; Android is pretty good; Mac is nearly unusable and Desktop has no accessibility whatsoever.

**Comment by MarcAnt01** [comment]
@AlexandrKozlovskiy this is not the right place to ask

**Comment by makisukurisu** [comment]
I guess this problem was fixed, and others either got a ticket, or fixed as well. If you have any problems - please, consider creating a new issue for it/comment there.
